apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -205,8 +205,8 @@ class ProgressBubble extends React.Component {
         <ProgressPill
           levels={[level]}
           text={i18n.unpluggedActivity()}
-          fontSize={16}
```

Comment:
hmm... these `ProgressBubbles` and `ProgressPill` components are used in more places than the header (the progress tab, legends, standards overview).  I think this style change should be fine, but I'm going to double check...

Reply:
👍  These components used in bubble choice, progress tab, teacher panel, etc all look good! 🎉

Reply:
Thank you for testing all of those cases!